src/main/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefApiApplication.java

```diff
@@ -6,16 +6,12 @@
 import org.springframework.cloud.openfeign.EnableFeignClients;
 import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import org.springframework.jms.annotation.EnableJms;
-import org.springframework.retry.annotation.EnableRetry;
 import uk.gov.hmcts.reform.idam.client.IdamApi;
 
 @EnableJpaAuditing
 @EnableJpaRepositories
 @SpringBootApplication
 @EnableCaching
-@EnableJms
-@EnableRetry
```

Comment:
The Azure library provides a separate mechanism for the retry. I have added it in MessageConfig.java